### PR TITLE
PLATFORM-1860 fixes for pylint

### PR DESCRIPTION
**Pull request by pguinard-public-com**
Refactoring fixes for pylint, this does not enable in circleci yet. That will be done in https://github.com/matadorapp/universal-static/pull/7 after getting 2 more issues resolved.

### CODE REVIEW

File: edge_router_public_com/requirements-dev.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 autopep8==2.1.*
 pytest-cov==5.0.*
 pylint==3.1.*
+pylint-junit==0.3.*
```
Comment (pguinard-public-com) [inline edge_router_public_com/requirements-dev.txt:4]: Requires __init__.py to be present.

File: edge_router_public_com/tests/test_mobile_headers.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+from helper_generate_cloudfront_request import generate_cloudfront_request
+from lambda_function import lambda_handler
+
+headers_mobile = {
```
Comment (pguinard-public-com) [inline edge_router_public_com/tests/test_mobile_headers.py:4]: These tests were missed in a git add.

Comment (nilspublic) [reply edge_router_public_com/tests/test_mobile_headers.py:4]: I would add this in it's own PR #9

**Comment by nilspublic** [comment]
I moved the workdir too for a simpler run in #10

**Comment by pguinard-public-com** [comment]
No changes after rebase.

**Pull request closed without merging**